libs/libcommon/src/libcommon/utils.py

```diff
@@ -65,6 +65,15 @@ class JobResult(TypedDict):
     output: Optional[JobOutput]
 
 
+class SplitHubFile(TypedDict):
```

Comment:
what's the idea of having a `Split` prefix? to reflect that the item must have a non-empty split?